src/main/java/sorald/processor/CollectionsEmptyConstantsProcessor.java

```diff
@@ -0,0 +1,33 @@
+package sorald.processor;
+
+import java.util.Collections;
+import sorald.annotations.ProcessorAnnotation;
+import spoon.reflect.code.*;
+import spoon.reflect.declaration.CtMethod;
+import spoon.reflect.declaration.CtType;
+
+@ProcessorAnnotation(
+        key = 1596,
+        description =
+                "\"Collections.EMPTY_LIST\", \"EMPTY_MAP\", and \"EMPTY_SET\" should not be used")
+public class CollectionsEmptyConstantsProcessor extends SoraldAbstractProcessor<CtFieldAccess<?>> {
+
+    @Override
+    protected void repairInternal(CtFieldAccess<?> element) {
+        CtType<?> collectionsType = getFactory().Class().get(Collections.class);
+
+        CtMethod<?> methodToBeCalled;
+        if (element.getVariable().getSimpleName().equals("EMPTY_LIST")) {
```

Comment:
On a side note, this violates the rule you implemented a processor for in #562  :)

(sorald-buildbreaker does not catch it because it uses Sorald 0.3.0, which doesn't have that processor)

Reply:
:X